File: 백준/Bronze/6980. Train Swapping/Train Swapping.py
import sys
input=lambda:sys.stdin.readline().rstrip()
for _ in range(int(input())):
    n,rlt=int(input()),0
    arr=[*map(int,input().split())]
    for i in range(n-1):
        for j in range(i+1,n):
            if arr[i]>arr[j]:
                rlt+=1
    print("Optimal train swapping takes %d swaps." % rlt)

File: 백준/Bronze/6980. Train Swapping/Train Swapping.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    int t, n, rlt;
    cin >> t;
    for (int i = 0; i < t; i++) {
        cin >> n;
        vector<int> arr(n);
        for (int j = 0; j < n; j++)
            cin >> arr[j];
        rlt = 0;
        for (int j = 0; j < n - 1; j++) {
            for (int k = j + 1; k < n; k++) {
                if (arr[j] > arr[k])
                    rlt += 1;
            }
        }
        cout << "Optimal train swapping takes " << rlt << " swaps.\n";
    }
    return 0;
}